docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,18 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. Add a functional test that requires the newly added static file to be delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :linenos:
```

Comment:
All tutorial pages print the complete file, even if only small modifications are made in respect to previous versions of the same file. This said, do you suggest to break with this patterns and only show a fragment of `tests.py` here?